packages/xdl/src/tools/UpdateVersions.js

```diff
@@ -80,10 +80,3 @@ export async function updateAndroidApk(s3Client: any, pathToApp: string, appVers
   versions['androidUrl'] = `https://d1ahtucjixef4r.cloudfront.net/Exponent-${appVersion}.apk`;
   await Versions.setVersionsAsync(versions);
 }
-
-export async function updateTurtleVersionAsync(sdkVersion: string, platform: string) {
```

Comment:
👍 

I've marked this as a breaking change (=> semver major version bump for XDL) because this function has been removed.